### PR TITLE
Milestone #5 for Magic Books Inc. Bookstore App

### PR DESCRIPTION
**Pull request by jadx2**
For this milestone, the styling of the website was implemented. Sass was used as a CSS preprocessor. The design followed the one given in Zeplin. Just a minor difference. Since it was required to show the book's author, a field in the form was created to handle that. It goes through Redux too as part of the book object.

### CODE REVIEW

File: src/styles/_book.scss [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      height: 1.125rem;
+      margin: 0 2.438rem 0 0;
+      opacity: 0.5;
+      font-family: Montserrat, sans-serif;
+      font-size: 0.875rem;
+      font-weight: bold;
+      font-stretch: normal;
+      font-style: normal;
+      line-height: normal;
+      letter-spacing: normal;
```
Comment (e71az) [inline src/styles/_book.scss:25]: You are repeating most of these rules 4 times, please apply the same rule to multiple elements just once, article for support: [good css practices](https://speckyboy.com/good-bad-css-practices/). (Ignore the ones that actually change every time)

File: src/styles/_book.scss [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    .book-title {
+      height: 1.813rem;
+      margin: 0.188rem 20.188rem 0 0;
+      font-family: 'Roboto Slab', sans-serif;
+      font-size: 1.375rem;
+      font-weight: bold;
+      font-stretch: normal;
+      font-style: normal;
+      line-height: normal;
+      letter-spacing: -0.2px;
```
Comment (e71az) [inline src/styles/_book.scss:38]: Repeated css rules.

File: src/styles/_book.scss [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    .book-author {
+      height: 1.188rem;
+      margin: 0.25rem 0 0;
+      font-family: 'Roboto Slab', sans-serif;
+      font-size: 0.875rem;
+      font-weight: 300;
+      font-stretch: normal;
+      font-style: normal;
+      line-height: normal;
+      letter-spacing: normal;
```
Comment (e71az) [inline src/styles/_book.scss:51]: Repeated css rules.

File: src/styles/_book.scss [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+      background-color: $white;
+      outline: none;
+      cursor: pointer;
+      font-family: 'Roboto Slab', sans-serif;
+      font-size: 0.875rem;
+      font-weight: 300;
+      font-stretch: normal;
+      font-style: normal;
+      line-height: normal;
+      letter-spacing: normal;
```
Comment (e71az) [inline src/styles/_book.scss:68]: Repeated css rules.

**Review by e71az — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️

Hello @jadx2,

Good job so far, I really liked the way that you handled the dynamic percentage number!

Your page is almost perfect, please just implement these simple required changes before getting it approved:

- [ ] [Black user icon](https://prnt.sc/1aym4yu) please change this icon to blue so we can comply with the "Exactly" keyword used by Microverse regarding the design given. Everything else looks perfect!
- [ ] Please refactor your CSS styles to prevent the same repetitive rule bad practice from happening as explained in this article: [css bad practices](https://speckyboy.com/good-bad-css-practices/) (Check the comments that I left in this review so you can see which ones I am referring to).

Please correct those issues and send another review request, good luck, and well done so far!

**Review by oloomoses — APPROVED** [review]
## Congratulations, Milestone Approved. :muscle: :tada: :clap: 
**Hello @jadx2,
Great job working on the changes suggested by @e71az.
You have passed all the requirements for this milestone :muscle:**

## Optional
- [ ] Kindly update the live version to match the local one

Feel free to merge and close this pull request.

**I wish you the best in your next project and see you at the top :rocket:** 
#### Happy Coding
_Slack: Oloo Moses_

**Pull request merged**